client/src/styles/PortfolioFilters.scss

```diff
@@ -368,6 +369,7 @@
                 border: 1px solid $justfix-black;
                 padding: 0.8rem 0.6rem;
                 margin-bottom: 1.1rem;
+                letter-spacing: 0.02em;
```

Comment:
what is the base `font-size` the `em` is referencing here?

Reply:
it's referring to the font size of the element, rather than the base. And it's 14px (1.4rem), which is set in the multi-select component.